.gitignore

```diff
@@ -33,6 +33,8 @@
 .rubocop-ruby.yml
 .rubocop.yml
 
+.idea/
```

Comment:
Developer-specific files/folders should not be here. How about using local gitignore?